src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java

```diff
@@ -221,4 +221,6 @@
     public static final String ERROR_DOCUMENT_SUB_ACCOUNT_GLOBAL_NEW_SUB_ACCOUNT_EXISTS = "error.document.subAccountGlobal.newSubAccountExists";
     public static final String ERROR_DOCUMENT_SUB_ACCOUNT_GLOBAL_CG_AND_NON_CG_MIX = "error.document.subAccountGlobal.cgAndNonCgMix";
     public static final String ERROR_DOCUMENT_SUB_ACCOUNT_GLOBAL_CG_EXPENSE_REQUIRED_SECTION = "error.document.subAccountGlobal.cgExpenseRequiredSection";
+    
+    public static final String ERROR_DOCUMENT_CONTRACT_GRANT_INVOICE_DOCUMENT_PRORATE_NO_AWARD_BUDGET_TOTAL = "error.document.contractGrantInvoiceDocument.prorate.no.award.buget.total";
```

Comment:
Minor typo in the constant's value; should be changed from "...award.buget.total" to "...award.budget.total"

Also, some of the similar error message keys in KFS have a prefix of "error.document.contractsGrantsInvoice" instead. Perhaps it would be helpful to follow that convention with this message key.

Reply:
ok